<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureGraphics</class>
 <widget class="QWidget" name="ConfigureGraphics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Graphics</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="toggle_hw_renderer">
          <property name="text">
           <string>Enable hardware renderer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="toggle_shader_jit">
          <property name="text">
           <string>Enable shader JIT</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="toggle_vsync">
          <property name="text">
           <string>Enable V-Sync</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Limit Speed Percent</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="frame_limit">
            <property name="suffix">
             <string>%</string>
            </property>
            <property name="maximum">
             <number>9999</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Internal Resolution:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="resolution_factor_combobox">
            <item>
             <property name="text">
              <string>Auto (Window Size)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Native (400x240)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2x Native (800x480)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>3x Native (1200x720)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4x Native (1600x960)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>5x Native (2000x1200)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>6x Native (2400x1440)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>7x Native (2800x1680)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>8x Native (3200x1920)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>9x Native (3600x2160)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>10x Native (4000x2400)</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QFrame" name="tex_cache_size_frame">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; text-decoration: underline;&quot;&gt;Cache Size&lt;/span&gt;&lt;/p&gt;&lt;p&gt;This options sets the number of scaled or custom textures that is cached. Changing this option will reduce lag spikes when when game reuses a texture that has already been upscaled.&lt;/p&gt;&lt;p&gt;Note this option is RAM heavy!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_13">
             <property name="toolTip">
              <string/>
             </property>
             <property name="text">
              <string>Cache Size</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="tex_cache_size">
             <property name="toolTip">
              <string/>
             </property>
             <property name="suffix">
              <string> Textures</string>
             </property>
             <property name="prefix">
              <string/>
             </property>
             <property name="minimum">
              <number>100</number>
             </property>
             <property name="maximum">
              <number>500</number>
             </property>
             <property name="singleStep">
              <number>10</number>
             </property>
             <property name="value">
              <number>100</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Texture Scaling</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="text">
         <string>Scaling Mode</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="indent">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSlider" name="tex_scale_factor">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>6</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
        <property name="invertedControls">
         <bool>false</bool>
        </property>
        <property name="tickPosition">
         <enum>QSlider::NoTicks</enum>
        </property>
        <property name="tickInterval">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="tex_scale_mode">
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bicubic</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>xBRz</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hybrid</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Scale</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>x</string>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="indent">
           <number>-1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="tex_scale_factor_lbl">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>66</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="tex_scale_depos">
        <property name="text">
         <string>Deposterize</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>label_3</zorder>
     <zorder>label_4</zorder>
     <zorder>tex_scale_factor</zorder>
     <zorder>tex_scale_mode</zorder>
     <zorder>tex_scale_depos</zorder>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Custom Textures</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QCheckBox" name="tex_dump">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Dump Textures</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="tex_load">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Load Custom Textures</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="tex_dump_dir">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="cursor">
         <cursorShape>IBeamCursor</cursorShape>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="tex_load_browse">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="tex_load_dir">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="tex_dump_browse">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="layoutBox">
     <property name="title">
      <string>Layout</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label1">
            <property name="text">
             <string>Screen Layout:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="layout_combobox">
            <item>
             <property name="text">
              <string>Default</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Single Screen</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Large Screen</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Side by Side</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="swap_screen">
          <property name="text">
           <string>Swap Screens</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>toggle_hw_renderer</tabstop>
  <tabstop>toggle_shader_jit</tabstop>
  <tabstop>toggle_vsync</tabstop>
  <tabstop>frame_limit</tabstop>
  <tabstop>resolution_factor_combobox</tabstop>
  <tabstop>tex_cache_size</tabstop>
  <tabstop>tex_scale_mode</tabstop>
  <tabstop>tex_scale_depos</tabstop>
  <tabstop>tex_scale_factor</tabstop>
  <tabstop>tex_dump</tabstop>
  <tabstop>tex_dump_dir</tabstop>
  <tabstop>tex_dump_browse</tabstop>
  <tabstop>tex_load</tabstop>
  <tabstop>tex_load_dir</tabstop>
  <tabstop>tex_load_browse</tabstop>
  <tabstop>layout_combobox</tabstop>
  <tabstop>swap_screen</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>toggle_hw_renderer</sender>
   <signal>toggled(bool)</signal>
   <receiver>resolution_factor_combobox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>519</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tex_dump</sender>
   <signal>toggled(bool)</signal>
   <receiver>tex_dump_dir</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tex_load</sender>
   <signal>toggled(bool)</signal>
   <receiver>tex_load_dir</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>347</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tex_dump</sender>
   <signal>toggled(bool)</signal>
   <receiver>tex_dump_browse</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>522</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tex_load</sender>
   <signal>toggled(bool)</signal>
   <receiver>tex_load_browse</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>352</y>
    </hint>
    <hint type="destinationlabel">
     <x>522</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tex_scale_factor</sender>
   <signal>valueChanged(int)</signal>
   <receiver>tex_scale_factor_lbl</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
